docs/architecture/decisions/0002-export-asrsnippet-metadata-for-snippet-metrics-processing.md

```diff
@@ -0,0 +1,46 @@
+# 2. Export ASRSnippet Metadata for Snippet Metrics Processing
+
+Date: 2019-02-25
+
+## Status
+
+Accepted
+
+## Context
+
+Data Engineers are building performance dashboards for Snippets using the
+metrics we collect from Firefox Telemetry. Telemetry pings include only basic
+information about the Snippet, like Snippet ID.
+
+For better to understand ad more complete dashboards, we want to enhance the
```

Comment:
can you fix this sentence?

        For better to understand ad more complete dashboards

Reply:
done, thanks!